src/main/kotlin/io/github/loxygen/aimlessbot/lib/client/Client.kt

```diff
@@ -1,92 +1,70 @@
-        jdaBuilder.setActivity(Activity.of(Activity.ActivityType.DEFAULT, "hennlo world"))
-        jdaBuilder.addEventListeners(Client)
-        discordClient = jdaBuilder.build()
+   /**
+    * コマンドを実行する
+    * @param token Discordのトークン
+    */
+   fun launch(token: String, clientSettingInfo: ClientSettingInfo) {
 
-    }
+      val jdaBuilder = JDABuilder.createDefault(token)
+      jdaBuilder.setActivity(Activity.of(Activity.ActivityType.DEFAULT, "hennlo world"))
+      jdaBuilder.addEventListeners(Client)
+      discordClient = jdaBuilder.build()
 
-    override fun onReady(event: ReadyEvent) {
-        println("ready confirmed!!!")
-        val channel = discordClient.getTextChannelById(695976154779222047) ?: error("sry where is channel")
-        channel.sendMessage("***†Flisan Aimless Bot Ready†***").queue()
-    }
+      loggingChannelId = clientSettingInfo.loggingChannelId
+      userIdsWhiteList = clientSettingInfo.botIdsWhiteList
 
-    override fun onMessageReceived(event: MessageReceivedEvent) {
+   }
 
-        if (event.author.isBot) {
-            return
-        }
+   override fun onReady(event: ReadyEvent) {
+      val channel = discordClient.getTextChannelById(
+         loggingChannelId
+      )
+      if (channel == null) {
+         println("--- Setting Error: Logging Channel ID ---")
+         println("A channel which has Id $loggingChannelId doesn't exist!")
+         println("Bot will exit with exit code 1.")
+         exitProcess(1)
+      }
+      channel.sendMessage("***†Flisan Aimless Bot Ready†***").queue()
+   }
 
-        var rawText = event.message.contentDisplay
-        var commandExecutorList: List<CommandExecutor> = prefixlessCommandExecutor
-        val prefixness = event.message.contentDisplay.startsWith(PREFIX)
-        if(prefixness) {
-            rawText = event.message.contentDisplay.substring(PREFIX.length)
-            commandExecutorList = commandExecutors
-        }
-        val commandArgs = rawText.split(" ")
+   override fun onMessageReceived(event: MessageReceivedEvent) {
 
-        for(command in commandExecutorList) {
-            val isCommandMatched = (prefixness && commandArgs[0] == command.identify) ||
-                                           (!prefixness && commandArgs[0].indexOf(command.identify) != -1)
-            if (isCommandMatched) {
-                when(command.parseCommand(commandArgs.subList(1, commandArgs.size), event)) {
-                    CommandResult.SUCCESS -> {}
-                    CommandResult.FAILED -> {event.channel.sendMessage("ズサーッ！(コマンドがコケた音)").queue()}
-                    CommandResult.INVALID_ARGUMENTS -> {event.channel.sendMessage("引数がおかしいみたいです").queue()}
-                    CommandResult.UNKNOWN_COMMAND -> {event.channel.sendMessage("そのサブコマンド is 何").queue()}
-                }
-                return
-            }
-        }
+      if ((event.author.isBot && !userIdsWhiteList.contains(event.author.idLong)) ||
```

Comment:
`contains` は `in` が演算子オーバーロードしています。
```suggestion
      if ((event.author.isBot && event.author.idLong !in userIdsWhiteList) ||
```